includes/class-wc-abstract-google-analytics-js.php

```diff
@@ -276,12 +276,14 @@ public function get_formatted_order( $order ): array {
 		}
 
 		return array(
-			'totals' => array(
-				'currency_code'       => $order->get_currency(),
-				'total_price'         => $this->get_formatted_price( $order->get_total() ),
-				'currency_minor_unit' => wc_get_price_decimals(),
-			),
-			'items'  => array_map(
+			'id'                  => $order->get_id(),
+			'affiliation'         => get_bloginfo( 'name' ),
+			'currency_code'       => $order->get_currency(),
+			'currency_minor_unit' => wc_get_price_decimals(),
+			'tax_total'           => $this->get_formatted_price( $order->get_total_tax() ),
+			'shipping_total'      => $this->get_formatted_price( $order->get_total_shipping() ),
+			'total_price'         => $this->get_formatted_price( $order->get_total() ),
+			'items'               => array_map(
```

Comment:
Thanks, that makes perfect sense. I was struggling with where to place the totals since I didn't want to repeat currency_code and currency_minor_unit for three different totals. But this is a nice way to combine it.